packages/clay-layout/src/Col.tsx

```diff
@@ -0,0 +1,88 @@
+/**
+ * SPDX-FileCopyrightText: © 2020 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import classNames from 'classnames';
+import React from 'react';
+
+type TColSize = boolean | number | 'auto';
+
+interface IProps extends React.HTMLAttributes<HTMLDivElement> {
+	/**
+	 * Position in which to align column
+	 */
+	align?: 'start' | 'center' | 'end';
```

Comment:
I'm still unsure about `align`... do we also have usages for this? I'm fine to keep it if you think it's safe and useful.